docs/layers/arc-layer.md

```diff
@@ -8,14 +8,16 @@
 
 The Arc Layer renders raised arcs joining pairs of source and target points,
 specified as latitude/longitude coordinates.
+[Source](https://github.com/uber/deck.gl/tree/master/src/layers/core/arc-layer)
```

Comment:
Not important, but having the source easily available right after the description of the layer will encourage people to look into it more than if it was at the very bottom of the docs after the props.

It takes very little place and is a good entry point if you want to start building your own layer or extending existing ones.

I guess if three.js did it, it could be fine, but I don't think it would be in our case.